### PR TITLE
chore(test): adding snapshot tests for metric factories

### PR DESCRIPTION
**Pull request by voho**

### CODE REVIEW

File: test/monitoring/aws/lambda/metrics.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { LambdaMetricFactory } from '../../../../src/monitoring/aws/lambda/metrics';
+
+const DummyTableName = 'DummyTableName';
```
Comment (ayush987goyal) [inline test/monitoring/aws/lambda/metrics.test.ts:3]: This should be `DummyLambdaName`

File: test/monitoring/aws/lambda/metrics.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test('snapshot test: metricThrottles', () => {
+  // GIVEN
+  const unitToTest = new LambdaMetricFactory();
+
+  // WHEN
+  const metric = unitToTest.metricThrottles(DummyTableName);
+
+  // THEN
+  expect(metric).toMatchSnapshot();
```
Comment (ayush987goyal) [inline test/monitoring/aws/lambda/metrics.test.ts:46]: Do you think it is better to also add explicit assertions, along with snapshots, for the most important properties? In this case that would probably be the `metricName` and `dimensions`. Thoughts?

File: test/monitoring/aws/lambda/metrics.test.ts (hunk 1 above)
Comment (voho) [reply test/monitoring/aws/lambda/metrics.test.ts:3]: Nice catch.

File: test/monitoring/aws/lambda/metrics.test.ts (hunk 2 above)
Comment (voho) [reply test/monitoring/aws/lambda/metrics.test.ts:46]: I do not see any major reason not to. Name and dimensions sounds as a good minimum, the snapshot can take care of the rest (color, labels mainly).

**Review by ayush987goyal — APPROVED** [review]

**Pull request merged**